services/oauth/v2/constants.go

```diff
@@ -0,0 +1,14 @@
+package v2
+
+const (
+	DestKey              ContextKey = "destination"
+	SecretKey            ContextKey = "secret"
+	RudderFlow_Delivery  RudderFlow = "delivery"
+	RudderFlow_Delete    RudderFlow = "delete"
+	DeleteAccountIdKey              = "rudderDeleteAccountId"
+	DeliveryAccountIdKey            = "rudderAccountId"
+
+	AuthStatusInactive = "inactive"
```

Comment:
GO's idiomatic naming convention. More about it in [here](https://go.dev/doc/effective_go#mixed-caps).

```suggestion
	RudderFlowDelivery  RudderFlow = "delivery"
	RudderFlowDelete    RudderFlow = "delete"
	DeleteAccountIDKey              = "rudderDeleteAccountId"
	DeliveryAccountIDKey            = "rudderAccountId"

	AuthStatusInActive = "inactive"
```